osu.Game/Screens/Play/HUD/GameplayAccuracyCounter.cs

```diff
@@ -1,20 +1,64 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-#nullable disable
-
+using System.ComponentModel;
 using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Game.Configuration;
 using osu.Game.Graphics.UserInterface;
 using osu.Game.Rulesets.Scoring;
 
 namespace osu.Game.Screens.Play.HUD
 {
     public abstract partial class GameplayAccuracyCounter : PercentageCounter
     {
-        [BackgroundDependencyLoader]
-        private void load(ScoreProcessor scoreProcessor)
+        [SettingSource("Accuracy display mode", "Which accuracy mode should be displayed.")]
+        public Bindable<AccuracyDisplayMode> AccuracyDisplay { get; } = new Bindable<AccuracyDisplayMode>();
+
+        [Resolved]
+        private ScoreProcessor scoreProcessor { get; set; } = null!;
+
+        protected override void LoadComplete()
         {
-            Current.BindTo(scoreProcessor.Accuracy);
+            base.LoadComplete();
+
+            AccuracyDisplay.BindValueChanged(mod =>
+            {
+                Current.UnbindBindings();
+
+                switch (mod.NewValue)
+                {
+                    case AccuracyDisplayMode.Standard:
+                        Current.BindTo(scoreProcessor.Accuracy);
+                        break;
+
+                    case AccuracyDisplayMode.MinimumAchievable:
+                        Current.BindTo(scoreProcessor.MinimumAccuracy);
+                        break;
+
+                    case AccuracyDisplayMode.MaximumAchievable:
+                        Current.BindTo(scoreProcessor.MaximumAccuracy);
+                        break;
+                }
+            }, true);
+
+            // if the accuracy counter is using the "minimum achievable" mode,
+            // then its initial value is 0%, rather than the 100% that the base PercentageCounter assumes.
+            // to counteract this, manually finish transforms on DisplayedCount once after the initial callback above
+            // to stop it from rolling down from 100% to 0%.
+            FinishTransforms(targetMember: nameof(DisplayedCount));
+        }
+
+        public enum AccuracyDisplayMode
+        {
+            [Description("Standard")]
+            Standard,
+
+            [Description("Maximum achievable")]
+            MaximumAchievable,
+
+            [Description("Minimum achievable")]
+            MinimumAchievable
```

Comment:
@bdach thoughts on making this something like "Count up" and "Count down"? It might be marginally more understandable to the end user.

Reply:
I dunno, that seems worse to me. "Count up" or "down" relative to what?

Reply:
To be honest, I'm not sure if anyone would actually want "minimum achievable" as a display mode. So maybe removing that then having "Current" of "Max achievable" as the only two options?

Reply:
> To be honest, I'm not sure if anyone would actually want "minimum achievable" 

Its the primary ( only? ) display mod in atleast some DDR games so theres def precedent for it no?  i can see some people appreciating it as an option idk

Reply:
Fair enough. I guess we'll go with the current terminology and see what the user reception is like.